### ISSUE TITLE
some small issues with admin panel in 2.1beta

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#390]
```
1. Capabilities is mispelt
2. The Client Apps overlaps with the main bar and is not centered in Chrome
3. When launching the orderfulfilment_customforms, as I start the first
workitem in the default worklist handler, the custom form appears. So far
so good. But now if I try to upload a file to fill out the form with data,
an exception is raised.
```

Original issue reported on code.google.com by `marcello...@gmail.com` on 4 Jun 2010 at 10:39

**Comment by yawlfoundation** [comment]
```
1. Do'h! Must've been a search  replace thing. Fixed now.
2. Do you mean the menu button?
3. Any details - stacktrace etc?
```

Original comment by `yawl.mic...@gmail.com` on 4 Jun 2010 at 11:43
* Changed state: **Started**

**Comment by yawlfoundation** [comment]
```
Point 2 is also a problem in Firefox 3.5, but not in Firefox 3.0 (at least on 
Ubuntu). There are also other rendering issues, which aren't just restricted to 
the admin panel:

-- the version number text overlaps the main areas on some screens

-- warning boxes don't scale vertically properly (see renderProb3_2-1.png)

These all occur on Firefox 3.5. The last of these also occurs on Firefox 3.0.

Would suggest this was higher than Low priority given the extent of the issues 
(even though they are 'just' cosmetic).
```

Original comment by `monsieur...@gmail.com` on 14 Jul 2010 at 3:51

Attachments:
* [renderingProb1_2-1.png](https://storage.googleapis.com/google-code-attachments/yawl/issue-391/comment-2/renderingProb1_2-1.png)
* [renderingProb2_2-1.png](https://storage.googleapis.com/google-code-attachments/yawl/issue-391/comment-2/renderingProb2_2-1.png)
* [renderingProb3_2-1.png](https://storage.googleapis.com/google-code-attachments/yawl/issue-391/comment-2/renderingProb3_2-1.png)
* [renderingProb4_2-1.png](https://storage.googleapis.com/google-code-attachments/yawl/issue-391/comment-2/renderingProb4_2-1.png)

**Comment by yawlfoundation** [comment]
```
Tweaked the message panel layout and auto-sizing metrics, resolving problem 3. 

FF 3.5 on Linux had some widely reported rendering problems that assumedly were 
the cause of the other layout problems. These issues were corrected in 3.6.

Have fully tested current code version in: FF 3.6 (Win, OSX, Ubuntu), Safari 
4.1 (OSX), Chrome 5.0 (Win, OSX, Ubuntu), Opera 10.6 (OSX, Ubuntu) and IE 6, 7, 
8 - all screens and dialogs render correctly on all platform/browser combos.
```

Original comment by `yawl.mic...@gmail.com` on 21 Jul 2010 at 6:50
* Changed state: **Fixed**
